integration-test/helpers/pages.js

```diff
@@ -649,7 +649,7 @@ export function emailAutofillPage (page) {
             const input = await page.locator(selectors.identity)
             if (checking === 'style') {
                 const style = await input.getAttribute('style')
-                expect(style).toBeNull()
+                expect(style).toBeFalsy()
```

Comment:
When we remove the styles the end result is `''` rather than `null`. This catches both. Let me know if you're aware of a reason we should actually be checking for `null`.

For context, we don't just remove the attribute altogether because the webpage may well be using the `style` attribute for their stuff, so we're careful only to remove the stuff we know we've added.